tianshou/utils/net/common.py

```diff
@@ -24,16 +24,28 @@ def miniblock(
     input_size: int,
     output_size: int = 0,
     norm_layer: Optional[ModuleType] = None,
+    norm_args: Optional[Union[Tuple[Any, ...], Dict[Any, Any]]] = None,
```

Comment:
Can we create a type called `ARGS_TYPE = Union[Tuple[Any, ...], Dict[str, Any], Sequence[Any]]` at beginning to reduce code duplication?